packages/gatsby-remark-images/README.md

```diff
@@ -58,6 +58,7 @@ plugins: [
 | `tracedSVG`             | `false` | Use traced SVGs for placeholder images instead of the "blur up" effect. Pass `true` for traced SVGs with the default settings (seen [here][3]), or an object of options to override the defaults. For example, pass `{ color: "#F00", turnPolicy: "TURNPOLICY_MAJORITY" }` to change the color of the trace to red and the turn policy to TURNPOLICY_MAJORITY. See [`node-potrace` parameter documentation][4] for a full listing and explanation of the available options.                                                                                                                                                        |
 | `loading`               | `lazy`  | Set the browser's native lazy loading attribute. One of `lazy`, `eager` or `auto`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
 | `disableBgImageOnAlpha` | `false` | Images containing transparent pixels around the edges results in images with blurry edges. As a result, these images do not work well with the "blur up" technique used in this plugin. As a workaround to disable background images with blurry edges on images containing transparent pixels, enable this setting.                                                                                                                                                                                                                                                                                                               |
+| `disableBgImage`        | `false` | Remove background image and its' inline style to prevent `Stylesheet too long` error on AMP, enable this setting.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
```

Comment:
```suggestion
| `disableBgImage`        | `false` | Remove background image and its' inline style. Useful to prevent `Stylesheet too long` error on AMP.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
```
Let's not focus on AMP use case here (but still mention it so people googling specific error can find this option). There can be plenty of other valid reason people might choose to disable bg images.

Reply:
It looks better. Thanks.